front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -397,9 +404,10 @@ export function AgentMessage({
         );
       }}
     >
-      <div ref={messageRef}>
+      <div>
         {renderMessage(agentMessageToRender, references, shouldStream)}
       </div>
+      <div ref={bottomRef} className="h-[50px]" />
```

Comment:
This might be a bit too high and leave too much of a big blank space.

Reply:
Why not using a `span` here as well?

Reply:
>This might be a bit too high and leave too much of a big blank space.

I updated according to what we discussed - setting the threshold to 1 and making the height of the bottom block lower. 

>Why not using a span here as well?

This should be a block in order to have the heigh. Span is an inline.